app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
I see where it could be confusing for someone who is not that familiar with embers Data Down Actions up paradigm. I could see why it seems odd when a `named` action is just sending the same `named` action up the line.

I don't think we *really* need to change the names between them. Though, if we were to want to have the names of the actions be different I would be more in favour of the route level actions being consistently called `save`. Something like:

*some component*
```javascript
savePost() {
  this.sendAction('save', post);
}
```

Then again that's just another opinion. Ultimately I am not sure if the naming is too important. The most important thing to convey is that we are not saving just part of the post I guess.